app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/provision_workflow/network.rb

```diff
@@ -40,15 +40,15 @@ def cloud_subnets(_options = {})
 
   # Fetch a list of security groups.
   # @param _options [void]
-  # @return [Hash<String, String>] Hash with ems_ref as key and name as value.
+  # @return [Hash<Integer, String>] Hash with id as key and name as value.
   def security_group_to_security_group(_options = {})
     cloud_network = field(:cloud_network)
     return {} if cloud_network.nil?
 
     ar_security_group = ar_ems.security_groups.select do |security_group|
       security_group.cloud_network.ems_ref == cloud_network
     end
-    string_dropdown(ar_security_group)
+    index_dropdown(ar_security_group)
```

Comment:
@GilbertCherrie do you have an example of what this looks like when it makes it to the payload? https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/provision/payload.rb#L25
I think you need this to be ems_ref not id

Reply:
It looks like currently we only support one being selected, if you want to support multiple it'd have to be a string array and we'd have to change the payload to do something like `security_groups => get_option(:security_groups).map { |ems_ref| {:id => ems_ref} }`